src/app/routes/getInitialData/cpsAsset/index.js

```diff
@@ -1,7 +1,15 @@
 import fetchData from '../utils/fetchData';
 import convertToOptimoBlocks from '#lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks';
+import applyTimestampRules from '#lib/utilities/preprocessor/rules/timestamp';
+import addIdsToBlocks from '#lib/utilities/preprocessor/rules/addIdsToBlocks';
+import applyBlockPositioning from '#lib/utilities/preprocessor/rules/blockPositioning';
 
-const preprocessorRules = [convertToOptimoBlocks];
+export const preprocessorRules = [
+  convertToOptimoBlocks,
+  applyTimestampRules,
```

Comment:
Is `applyTimestampRules` needed in this PR?

Reply:
Not needed, but i just used all the articles preprocessors